core_apps/institutes/templates/institutes/manage academicSection/academicSection.html

```diff
@@ -0,0 +1,96 @@
+{% extends "common/institute_layout.html" %} 
```

Comment:
Never use spaces in folder name. Always use underscore `_` if want to show seperate words.